README Corrections (#1709)
vuejs/eslint-plugin-vue: README.md

Be sure to read the [official ESLint guide](https://eslint.org/docs/developer-guide/working-with-rules) before you start writing a new rule.

To see what an abstract syntax tree (AST) of your code looks like, you may use [AST Explorer](https://astexplorer.net). After opening [AST Explorer](https://astexplorer.net), select `Vue` as the syntax and `vue-eslint-parser` as the parser.

> The default JavaScript parser must be replaced because [Vue.js single file components](https://v3.vuejs.org/guide/single-file-component.html#single-file-components) are written in a non-standard syntax. [`vue-eslint-parser`](https://github.com/vuejs/vue-eslint-parser) is a replacement parser that generates an enhanced AST with nodes that represent specific parts of the template syntax, as well as the contents of the `<script>` tag.

Reviewer: I prefer the previous term "not plain JavaScript", as it's much clearer than "non-standard syntax" (what standard?).

```suggestion
The default JavaScript parser must be replaced because [Vue.js single file components](https://v3.vuejs.org/guide/single-file-component.html#single-file-components) are not plain JavaScript. [`vue-eslint-parser`](https://github.com/vuejs/vue-eslint-parser) is a replacement parser that generates an enhanced AST with nodes that represent specific parts of the template syntax, as well as the contents of the `<script>` tag.
```

Author: I think "plain" is adequate here as well. My goal with the change to "non-standard syntax" was to give the user some keywords that would help them discover this page describing the SFC syntax specification: https://v3.vuejs.org/api/sfc-spec.html#intro

Perhaps there's another term we could use. "Unique" syntax? "Special" syntax? ".vue SFC" syntax?

"...written in the [SFC syntax](https://v3.vuejs.org/api/sfc-spec.html#intro)"

Reviewer: Maybe both?

"… single file components are not plain JavaScript, but a special file format."

Author: I'm noticing the word "custom" being used in the SFC syntax specification. I think if we replace "special" with "custom" in your example above, that makes the most sense to me.

> The default JavaScript parser must be replaced because [Vue.js single file components](https://v3.vuejs.org/guide/single-file-component.html#single-file-components) are not plain JavaScript, but a special file format. [`vue-eslint-parser`](https://github.com/vuejs/vue-eslint-parser) is a replacement parser that generates an enhanced AST with nodes that represent specific parts of the template syntax, as well as the contents of the `<script>` tag.

Reviewer: @chuntington I thought you suggested this?

```suggestion
The default JavaScript parser must be replaced because [Vue.js single file components](https://v3.vuejs.org/guide/single-file-component.html#single-file-components) are not plain JavaScript, but a custom file format. [`vue-eslint-parser`](https://github.com/vuejs/vue-eslint-parser) is a replacement parser that generates an enhanced AST with nodes that represent specific parts of the template syntax, as well as the contents of the `<script>` tag.
```

Author: Thank you!
